### ISSUE TITLE
This add-on does not seem to work anymore

### ISSUE THREAD

**Issue by jaruba** [pedroslopez/stremio-cuevana2esp#1]
It does not give any streams

**Comment by pedroslopez** [comment]
Yeah, it doesn't seem to be an issue with the add-on itself though, but rather with the site it's based on. The add-on doesn't support all the site's sources because of the way they load videos, and it seems the sources it _does_ support are having issues at the moment on the site itself. 

I'll check soon if I can integrate the other sources it provides or just switch the site where I'm getting the links from.
